### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by Chipx0** [server-status-project/server-status#96]
Is this project dead? I would like to see more features.

**Comment by Pryx** [comment]
I'm gonna be honest, unfortunately it pretty much is... I don't have much time keeping up with my studies and working simultaneously. I still want to do some much needed quality of life improvements to the codebase and update the translations now and then. We will see about the improvements in the summer, I might have more time then. Anyone is welcome to pick up the project in the meantime :)

**Comment by Pryx** [comment]
I kinda want to do a complete rewrite to incorporate all the features I wanted to have and at the same time to make it much more stable (I never expected the `mysqli_stmt::get_result()` problem). I don't expect returning to this project and understanding everything so it might be the better option. Migration would be a challenge though, but I would love to be able to migrate every existing installation via some upgrade script if that would be the case.

**Comment by Chipx0** [comment]
Ah i see, well if you do happen to rewrite the entire thing, I think adding some type of pinging system kind of like uptime robot would be a great addition.

**Comment by SigmaScorpii** [comment]
@Pryx I was thinking of picking this project up, just for solving a few issues.
That way, people can still use it, as I've seen a returning issue (#95 ), which isn't that hard to fix.

I won't be adding any new features.
Is it okay for you if I create a fork of it when I have some spare time?

**Comment by Pryx** [comment]
@SigmaScorpii please do! I will gladly accept any bugfix pull requests :slightly_smiling_face:

**Comment by thnilsen** [comment]
I've started looking at this project again as I've got a bit of time. Working on two features at the moment:
1. Updating PR #71 code to work from a queue table and get it included in the config routine.
2. Add grouping of services which I believe issue #90 is requesting

In addition I see that the #83 request for RSS should be easy to add to the subscriber menu created in PR #71. Will probably implemtent both RSS and Atom at the same time.
